domain/src/main/java/life/qbic/usermanagement/User.java

```diff
@@ -0,0 +1,143 @@
+package life.qbic.usermanagement;
+
+import java.util.UUID;
+import life.qbic.usermanagement.policies.*;
+
+public class User {
+
+  private String id;
+
+  private String fullName;
+
+  private String email;
+
+  private String encryptedPassword;
+
+  /**
+   * Creates a new user account, with a unique identifier to unambiguously match the user within
+   * QBiC's organisation.
+   *
+   * @param password the desired password for the user
+   * @param fullName the full name of the user
+   * @param email    the email address of the user
+   * @return the new user
+   * @since 1.0.0
+   */
+  public static User create(String password, String fullName, String email) throws UserException {
+    String uuid = String.valueOf(UUID.randomUUID());
```

Comment:
do we need to check if this id is already in use?

Reply:
a) Covered by tests
b) UUID version 4 collision probability is low:
https://en.wikipedia.org/wiki/Universally_unique_identifier#Version_4_(random)